koku/masu/database/trino_sql/gcp/get_invoice_month_dates.sql

```diff
@@ -0,0 +1,29 @@
+SELECT
+    min(usage_start_time),
+    max(usage_start_time)
```

Comment:
This is just a potential performance enhancement. Part of me is curious if we could return the year & month here, and remove the extra filters we are doing in 

- koku/masu/database/trino_sql/gcp/openshift/populate_daily_summary/1_resource_matching_by_cluster.sql

The only reason I call it out is cause the two queries are very very similar now.